insecure/corruptlibp2p/spammerGossipSub.go

```diff
@@ -0,0 +1,32 @@
+package corruptlibp2p
+
+import (
+	"github.com/libp2p/go-libp2p/core/peer"
+	pubsub "github.com/yhassanzadeh13/go-libp2p-pubsub"
+)
+
+type ControlMessage int
+
+// SpammerGossipSub is a wrapper around the GossipSubRouter that allows us to
+// spam the victim with junk control messages.
+type SpammerGossipSub struct {
+	router *pubsub.GossipSubRouter
+}
+
+func NewSpammerGossipSubRouter(router *pubsub.GossipSubRouter) *SpammerGossipSub {
+	return &SpammerGossipSub{
+		router: router,
+	}
+}
+
+// SpamIHave spams the victim with junk iHave messages.
+// msgCount is the number of iHave messages to send.
+// msgSize is the number of messageIDs to include in each iHave message.
```

Comment:
Add note in godoc mentioning msgSize is currently static.

Reply:
they are not static, we use the parameters: https://github.com/onflow/flow-go/blob/dc80931f84d4cfa3fb7cf770d0fb2a481f008df8/insecure/corruptlibp2p/fixtures.go#L31-L37

Reply:
I was referring to the underlying method used. Doesn't look like we currently use the message size and create them all with a size of 10. 

https://github.com/onflow/flow-go/blob/2cbea2a4f0df1643b72ea1b2c1c8ed4d9ec030e7/insecure/corruptlibp2p/fixtures.go#L42-L46